### ISSUE TITLE
Install command creates invalid .git/hooks/post-change if called explicitly

### ISSUE THREAD

**Issue by sascha-egerer** [captainhookphp/captainhook#189]
Calling `vendor/bin/captainhook install post-change` creates a hook file in `.git/hooks/post-change` which does not make sense as it is only a "virtual hook" by captainhook. So this one should better create the hooks `post-checkout` `post-merge `post-rewrite` or simply fail like if an invalid hook-name is given.

```
vendor/bin/captainhook install post-change
  'post-change' hook installed successfully
```

```
vendor/bin/captainhook install invalid

In Installer.php line 142:

  Invalid hook name 'invalid'


install [-c|--configuration [CONFIGURATION]] [-g|--git-directory [GIT-DIRECTORY]] [-f|--force] [-s|--skip-existing] [--move-existing-to [MOVE-EXISTING-TO]] [-b|--bootstrap [BOOTSTRAP]] [-m|--run-mode [RUN-MODE]] [-e|--run-exec [RUN-EXEC]] [-p|--run-path [RUN-PATH]] [--] [<hook>]
```

**Comment by sebastianfeldmann** [comment]
Thanks for the feedback. I changed the installer to make sure no virtual hook gets installed by accident.

Next release this will be fixed.
Thanks again.

**Comment by sebastianfeldmann** [comment]
Thanks for the feedback. I changed the installer to make sure no virtual hook gets installed by accident.

Next release this will be fixed.
Thanks again.

**Comment by sebastianfeldmann** [comment]
Just released version 5.11.1 including this fix.
